src/v2/view-builder/components/ChallengeWebauthnFooter.js

```diff
@@ -0,0 +1,98 @@
+import AuthenticatorFooter from './AuthenticatorFooter';
+import { $, _, loc, View } from 'okta';
+import { getBackToSignInLink } from '../utils/LinksUtil';
+import Link from './Link';
+import hbs from 'handlebars-inline-precompile';
+
+const CantVerifyInfoVerifyFlowTemplate = View.extend({
+  id: 'help-description-container',
+  className: 'help-description js-help-description',
+  template: hbs`
+    <div className="help-description js-help-description" id="help-description-container">
+      <h3>{{i18n code="oie.verify.webauthn.cant.verify.biometric.authenticator.title" bundle="login"}}</h3><br>
+      <p>{{i18n code="oie.verify.webauthn.cant.verify.biometric.authenticator.description1" bundle="login"}}</p><br>
+      <p>{{i18n code="oie.verify.webauthn.cant.verify.biometric.authenticator.description2" bundle="login"}}</p><br>
+      <h3>{{i18n code="oie.verify.webauthn.cant.verify.security.key.title" bundle="login"}}</h3><br>
+      <p>{{i18n code="oie.verify.webauthn.cant.verify.security.key.description" bundle="login"}}</p><br>
+    </div>
+  `,
+});
+
+const CantVerifyInfoOVEnrollmentFlowTemplate = View.extend({
```

Comment:
we generally call them view not `templates` .. `CantVerifyInfoOVEnrollmentFlowHelpLinkView`